package.json

```diff
@@ -27,14 +27,14 @@
     "coveralls": "npm run cover && istanbul-coveralls"
   },
   "dependencies": {
+    "arr-filter": "^1.1.1",
+    "arr-flatten": "^1.0.1",
+    "arr-map": "^2.0.0",
+    "array-each": "^0.1.1",
```

Comment:
I opened an issue on `array-each` to bump to 1.0 - ref https://github.com/jonschlinkert/array-each/issues/1

Hoping to get that bumped before merging this.